middleware/config/config.json

```diff
@@ -0,0 +1,9 @@
+{
+    "secret": "e32bd326eae57036be5c17f1bd6b9bd36b7e5b8e",
```

Comment:
We'll have to remove this and delete it from the backend too as it is now exposed in GitHub, can you make this read an environment variable instead?

Reply:
> What is this secret?
> Is it safe to publish this?
> @pradyu1
this value is used to JWT encryption. It can be changed. It isn't a GCP key. Yes we have made those changes